apollo/models/agent.py

```diff
@@ -16,6 +17,19 @@ class Agent(Base):
     oauth_client = relationship('OAuthClient', uselist=False,
                                 cascade="all, delete-orphan")
 
+    @property
+    def connection_status(self):
+        try:
+            return str(
+                WebSocketManager().connections[self.id].client_state).split(
+                    '.')[1].lower() is None
+        except KeyError:
+            return 'no connection available'
```

Comment:
I'd say this is 'disconnected'. I originally imagined the whole thing in either 'connected' or 'disconnected' but 'connecting' is a nice adddition.